firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionConnectionDetailsView.swift

```diff
@@ -0,0 +1,119 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+
+final class TrackingProtectionConnectionDetailsView: UIView {
+    private struct UX {
+        static let foxImageSize: CGFloat = 100
+        static let connectionDetailsLabelsVerticalSpacing: CGFloat = 12
+        static let connectionDetailsLabelBottomSpacing: CGFloat = 28
+        static let connectionDetailsStackSpacing = 15.0
+    }
+
+    private let connectionDetailsContentView: UIView = .build { view in
+        view.layer.cornerRadius = TPMenuUX.UX.viewCornerRadius
```

Comment:
Looks like you added the UX constants in this Details View. Should this `TPMenuUX` constants go inside that too?

Reply:
viewCornerRadius is used in many files.